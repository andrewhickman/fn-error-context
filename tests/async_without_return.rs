use fn_error_context::context;

#[context("context")]
async fn async_something() {
}

fn main() {
}
