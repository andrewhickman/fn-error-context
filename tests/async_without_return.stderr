error: function should return Result
 --> $DIR/async_without_return.rs:3:1
  |
3 | #[context("context")]
  | ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
